test/createAzureResource.test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { ResourceManagementClient } from 'azure-arm-resource';
+// tslint:disable-next-line:no-require-imports
+import WebSiteManagementClient = require('azure-arm-website');
+import { Site } from 'azure-arm-website/lib/models';
+import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
+import * as vscode from 'vscode';
+import { AzureTreeDataProvider, DialogResponses, TestAzureAccount, TestUserInput } from 'vscode-azureextensionui';
+import { ext } from '../src/extensionVariables';
+import { localize } from '../src/localize';
+import { FunctionAppProvider } from '../src/tree/FunctionAppProvider';
+import * as fsUtil from '../src/utils/fs';
+
+// This ensures that this will only run on nightly cron builds
```

Comment:
You're being too technology-specific here.  This determination should be based on something more directly tied to the intention, e.g. a TEST_TYPE environment variable or ENABLE_LONG_RUNNING_TESTS or something.  Then in travis.yml you can set that up if it's a cron build.  That'll make it much easier to enable when not on travis.